### ISSUE TITLE
[Accessibility] Workbench: Back button unresponsive to 'enter' key

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#1022]
**Describe the bug**
The back button on the workbench page can receive keyboard focus when tabbing through the page. However, when the button has focus and the Enter key is pressed, it does not perform its expected action of returning to the previous page. This issue impacts keyboard accessibility, as users who rely on keyboard navigation cannot use the back button functionality.

**To Reproduce**
1. Navigate to the workbench page
2. Use the Tab key to move focus to the back button
3. Press the Enter key

**Expected behavior**
Pressing Enter should activate the back button, returning the user to the previous page.

**Suggested investigation**
Check the event listener for the Enter key press on the back button.
Verify that the button's onClick event (or equivalent) is properly set up for keyboard interactions.

**Additional context**
The focus visibility of the button has been fixed in a previous update. This issue specifically relates to the button's lack of response to the Enter key.

**Comment by YashrajKupekar17** [comment]
I would like to solve this issue. Please assign it to me.

**Comment by birm** [comment]
Hi @YashrajKupekar17; just a reminder that we don't typically assign issues to people. If we did, we'd have a lot of orphaned issues/PRs. If you want to work on something, work on something and submit a PR! Some guidance for contributors is collected at http://camicroscope.org/contributing.html ; please let us know in a new issue if you think it's unclear!

That said, the [accessibility] issues raised by @nwanduka are much more likely to also be completed by them, as they're participating in an open source internship.
